File: Web/ClientApp/src/app/models/db/subdistrict.ts
import { Guid } from "guid-typescript";
import { EntityBase } from "../entityBase";

export class Subdistrict extends EntityBase {
    subdistrictCode: Guid;
    provinceCode: Guid;
    districtCode: Guid;
    subdistrictTh: string;
    subdistrictEn: string;
    postalCode: string;
    active: boolean;
    description: string;
}

File: Domain/Entities/DB/Subdistrict.cs
using System;

namespace Domain.Entities.DB
{
    public class Subdistrict : EntityBase
    {
        public Guid? SubdistrictCode { get; set; }
        public Guid? DistrictCode {  get; set; }
        public string SubdistrictTh {  get; set; }
        public string SubdistrictEn { get; set; }
        public string PostalCode { get; set; }
        public bool? Active { get; set; }
        public string Description { get; set; }
    }
}
